packages/expect/src/__tests__/matchers.property.test.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+const chalk = require('chalk');
+const chalkEnabled = chalk.enabled;
+const fc = require('fast-check');
+
+// settings for anything arbitrary
+const anythingSettings = {
+  withBoxedValues: true,
+  withMap: true,
+  withSet: false,

```

Comment:
Issue https://github.com/facebook/jest/issues/7975 have to be fixed before enabling the generation of `Set`

Reply:
Same for `Map`

Reply:
can you add that as a code comment?